packages/core/src/lib/session.ts

```diff
@@ -14,20 +15,30 @@ export const assignInteractionResults = async (
   // have to do it manually
   // refer to: https://github.com/panva/node-oidc-provider/blob/c243bf6b6663c41ff3e75c09b95fb978eba87381/lib/actions/authorization/interactions.js#L106
   const details = merge ? await provider.interactionDetails(ctx.req, ctx.res) : undefined;
-
+  const ts = dayjs().unix();
+  const mergedResult = {
+    // Merge with current result
+    ...details?.result,
+    ...result,
+  };
   const redirectTo = await provider.interactionResult(
     ctx.req,
     ctx.res,
     {
-      // Merge with current result
-      ...details?.result,
-      ...result,
+      ...mergedResult,
+      login: mergedResult.login
+        ? {
+            ...mergedResult.login,
+            // Update ts(timestamp) if the accountId has been set in result
+            ts: result.login?.accountId ? ts : mergedResult.login.ts,
```

Comment:
is there any specific reason we use `ts` over `timestamp`?

Reply:
<img width="832" alt="image" src="https://user-images.githubusercontent.com/12833674/200469924-39923762-de25-4c94-aecd-ea78dcb70b39.png">